system_test/helpers/aest_docker.erl

```diff
@@ -187,7 +188,12 @@ stop_node(NodeState) -> stop_node(NodeState, #{}).
 
 -spec stop_node(node_state(), stop_node_options()) -> node_state().
 stop_node(#{container_id := ID, hostname := Name} = NodeState, Opts) ->
-    aest_docker_api:stop_container(ID, Opts),
+    Timeout = maps:get(soft_timeout, Opts, ?EPOCH_STOP_TIMEOUT),
+    aest_docker_api:exec(ID, ["/home/epoch/node/bin/epoch", "stop"]),
```

Comment:
It might be good idea to factor this path (dir) out (e.g. as constant)  as it might change in future

Reply:
Or assume that workdir in Docker image is already the one from which bin/epoch can be run.